array/DNAm/analysis/EWAS/dmrcate.r

```diff
@@ -0,0 +1,123 @@
+#---------------------------------------------------------------------#
+##
+## Title: Regional analysis for case control within each cell type
+##
+## Purpose of script: identify regions of differential  DNA methylation for
+## schizophrenia vs controls for each cell type separately using DMRcate
+##
+##---------------------------------------------------------------------#
+
+#----------------------------------------------------------------------#
+# LOAD PACKAGES
+#----------------------------------------------------------------------#
+
+library(DMRcate)
+
+#----------------------------------------------------------------------#
+# DEFINE FUNCTIONS
+#----------------------------------------------------------------------#
+
+gg_color_hue <- function(n) {
+  hues = seq(15, 375, length = n + 1)
+  hcl(h = hues, l = 65, c = 100)[1:n]
+}
+
+#----------------------------------------------------------------------#
+# DEFINE PARAMETERS
+#----------------------------------------------------------------------#
+
+args<-commandArgs(trailingOnly = TRUE)
+dataDir <- args[1]
+cellType <- args[2]
+refPath <- args[3]
+
+normData<-file.path(dataDir, "3_normalised/normalised.rdata")
+cellCompData<-file.path(dataDir, "4_analysis", "EstimatedNeuralCellComposition.rdata")
```

Comment:
The job submission script doesn't currently either create the "4_analysis" directory, or run the lmWithinCT.r script to generate the "EstimatedNeuralCellComposition.rdata" object so these have to be manually created./run